spec/spec_helper.rb

```diff
@@ -2,5 +2,11 @@
 require "hanami/utils"
 require "hanami/devtools/unit"
 require "hanami/mailer"
+require "pry"
+require "pry-nav"
+
+Hanami::Mailer.configure do
```

Comment:
Can you please find an alternative to this? In `unstable` branch global configuration is gone. I will have hard times to port these tests. Thanks!